packages/validator-ajv8/src/validator.ts

```diff
@@ -202,17 +204,66 @@ export default class AJV8Validator<
    * @private
    */
   private transformRJSFValidationErrors(
-    errors: ErrorObject[] = []
+    errors: ErrorObject[] = [],
+    uiSchema?: UiSchema<T, S, F>
   ): RJSFValidationError[] {
     return errors.map((e: ErrorObject) => {
-      const { instancePath, keyword, message, params, schemaPath } = e;
+      const {
+        instancePath,
+        keyword,
+        params,
+        schemaPath,
+        parentSchema,
+        ...rest
+      } = e;
+      let { message } = rest;
       let property = instancePath.replace(/\//g, ".");
-      let stack = `${property} ${message}`.trim();
+
+      let stack: string | undefined = undefined;
+
       if ("missingProperty" in params) {
         property = property
           ? `${property}.${params.missingProperty}`
           : params.missingProperty;
+
+        const currentProperty: string = params.missingProperty;
+
+        const uiSchemaTitle = get(
+          uiSchema,
+          `${property.replace(/^\./, "")}.['ui:title']`
+        );
```

Comment:
Given that `title` could also be in the `ui:options` object, Do this instead:
```suggestion
        const uiSchemaTitle = getUiOptions<T, S, F>(get(uiSchema,`${property.replace(/^\./, "")}`)).title;
```